src/VisualStudio/LiveShare/Impl/AbstractClassificationsHandler.cs

```diff
@@ -28,6 +28,7 @@ internal abstract class AbstractClassificationsHandler : ILspRequestHandler<Clas
 
         public async Task<object[]> HandleAsync(ClassificationParams request, RequestContext<Solution> requestContext, CancellationToken cancellationToken)
         {
+            request.TextDocument.Uri = requestContext.ProtocolConverter.FromProtocolUri(request.TextDocument.Uri);
```

Comment:
wat.

Reply:
@CyrusNajmabadi This is a custom LSP request using the old liveshare APIs.  Unfortunately, liveshare doesn't do path conversion on these requests, so we need to convert the guest path to the host path ourselves.  

All of this is getting deleted next sprint as that's when classification support is coming to the LSP client (and path conversion will be handled automatically as well)